docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,328 @@
+# Prometheus Time Series Database migration
+This article provides a set of guidelines to migrate the time series database (TSDB from now on) from an existing Prometheus deployment to the Bitnami Kubernetes Production Runtime one.
```

Comment:
Sorry if this is a basic question but I think we should probably add some context for this scenario ie. why would a user want to perform this migration or what's the benefit/purpose in doing so?

Reply:
added, thank you